circle.yml

```diff
@@ -0,0 +1,12 @@
+dependencies:
+  pre:
+    - wget https://github.com/coreos/etcd/releases/download/v0.4.6/etcd-v0.4.6-linux-amd64.tar.gz
+    - tar xzvf etcd-v0.4.6-linux-amd64.tar.gz
+    - sudo cp etcd-v0.4.6-linux-amd64/etcd* /usr/local/bin/
+  override:
+    - 'rvm-exec 2.1.5 bundle install'
```

Comment:
Let's just let it use the circleci default. If it works in 1.9.3 it should work in ruby 2.x